xtrack/beam_elements/elements.py

```diff
@@ -723,6 +723,77 @@ def get_backtrack_element(self, _context=None, _buffer=None, _offset=None):
                               _buffer=_buffer, _offset=_offset)
 
 
+class ThickCombinedFunctionDipole(BeamElement):
```

Comment:
Let's go for several elements:

- Sbend
- Rbend
- Bend
- Quadrupole
- CombinedFunctionMagnet

Need to figure out what to do about edge effects